samples/iotc-sensors-gps/Kconfig

```diff
@@ -54,6 +61,18 @@ config USE_SENSOR_SIM
 	select SENSOR_SIM_TRIGGER
 	select SENSOR_SIM_STATIC_VALUES
 
+menuconfig THINGY_SENSORS
```

Comment:
I absolutely agree that we should use the conf file instead of this KConfig mess. For some reason the sensor drivers didn't get loaded and Thingy91 would fail to initialize the sensors at runtime. I logged a ticket #12 to revisit this.